# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h
# Commit: df07abc82381
# Full Hash: df07abc82381d2a3067b7de63867a0181f497519
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-26 09:13:23
# Regressor Bug: 1899503
# File Overlap Count: 2
# Description:
#   Bug 1899503 - P2 PRemoteWorkerDebuggerMangaer binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebuggerManager binding.
#   
#   PRemoteWorkerDebuggerManager is the IPC that handles the registration/unregistration of remote debuggers.
# ==============================================================================

diff -r 5d39af040a61 -r df07abc82381 dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h
--- a/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h	Tue Mar 25 14:33:27 2025 +0000
+++ b/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerParent.h	Tue Mar 25 14:33:27 2025 +0000
@@ -19,6 +19,9 @@
   NS_INLINE_DECL_THREADSAFE_REFCOUNTING(RemoteWorkerDebuggerManagerParent,
                                         final)
 
+  static RefPtr<RemoteWorkerDebuggerManagerParent> CreateForProcess(
+      mozilla::ipc::Endpoint<PRemoteWorkerDebuggerManagerChild>* aChildEp);
+
   RemoteWorkerDebuggerManagerParent();
 
   mozilla::ipc::IPCResult RecvRegister(